libs/database/src/lib/services/authentication.service.ts

```diff
@@ -523,4 +524,28 @@ export class AuthenticationService {
     });
     return accountActivation.save();
   }
+
```

Comment:
เพิ่ม comment ประมาณนี้สำหรับ gen document ด้วยคับ
 /**
   * this will assume that we already check otp is valid. this function will change current account password and delete otp then return newly change password account
   * @param {AccountDocument} account
   * @param {OtpDocument} otp
   * @param {string} newPassword
   * @returns {Promise<{AccountDocument}>}
   */